### PR TITLE
Refactor resource page and add the functionality to rate a resource

### PR DESCRIPTION
**Pull request by ristofer**
I started from Ronald branch and added the functionality to rate a resource starting with the UserRating component. It is working but has a bug. At the second click on the Rating component occurs the re-render.  Tomorrow I will ask Benjamin.

### CODE REVIEW

**Review by jjsantos01 — CHANGES_REQUESTED** [review]
Waiting for the fix of the bug in userRating component that doesn't update instantly.

**Comment by ristofer** [comment]
Changes are ready.

File: components/resources/CommentSection.jsx [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
-        <Comment
-          key={comment.id}
-          userName={comment.user.name}
-          postTime={comment.created_at}
-          content={comment.content}
-          userId={comment.user.id}
-        />
+        <Comment key={comment.id} comment={comment} />
       ))}
+      <NewResourceComment
```
Comment (jjsantos01) [inline components/resources/CommentSection.jsx:24]: I find it more user-friendly if the `new comment` button is above of the existing comments.

**Review by jjsantos01 — APPROVED** [review]
Nice refactoring!
I tested and worked ok.
I only left a comment about the order of the components in the comments section, nothing of importance.
Excellent job!

**Pull request merged**